html/semantics/embedded-content-0/media-elements/video_008.htm

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Change this to HTML5 Media Elements: 'media' attribute</title>
```

Comment:
Remove "Change this to " :-D

LGTM after fixing that. Feel free to squash the commits into one if you like. Please add a new comment if you want me to merge.